components/button/demo/loading.md

```diff
@@ -32,7 +32,7 @@ class App extends React.Component {
 
   render() {
     return (
-      <span>
+      <div>
```

Comment:
Any reason to change this?

Reply:
A span tag contains too much tags that is weird.